docs/template-vm.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: linux-vm-template
+  annotations:
+    description: "OCP kubevirt linux, template"
+    tags: "kubevirt,ocp,template,linux"
+objects:
+- apiVersion: kubevirt.io/v1alpha1
+  kind: OfflineVirtualMachine
+  metadata:
+    name: linux-vm-${NAME}
+  spec:
+    template:
+      metadata:
+      labels:
+      spec:
+        domain:
+          resources:
+            memory: ${MEMORY}
+            cpu: ${CPU}
+          os:
+            machine: pc-q35-rhel7.4.0
+          devices:
+            interface:
+              model:
+                type: virtio
+            disks:
+              - name: disk0
+                volumeName: disk0-pvc
+                disk:
+                  dev: vda
+                  bus: virtio
+  volumes:
+    - name: disk0-pvc
+      persistentVolumeClaim:
+        claimName: linux-vm-pvc-${NAME}
```

Comment:
Unlike the name of the `OfflineVirtualMachine`, the name of the claim, and the name of the persistent volume, can be calculated, with a prefix if you find it appropriate, because the user won't care about those names, actually we won't show them in the UI (I mean in the ManageIQ UI).